boards/FSM/Src/Io/Io_BrakePressureSensor.c

```diff
@@ -0,0 +1,18 @@
+#include "Io_Adc.h"
+
+float Io_BrakePressureSensor_GetPressurePsi(void)
```

Comment:
By the way, what's the part no. of the brake pressure? Out of curiosity

Reply:
I think it's worth documenting that within this function. I'd even go so far as to say that we should be labelling these `Io_CompanyNameModelNumber_` (for both the file and functions) so that it's very clear that this was designed for a very specific brake pressure sensor.

Reply:
@garethellis0  I added the sensor model to the javadoc for `Io_BrakePressureSensor_GetPressurePsi.h` for now. I asked Ken about renaming the file to what you suggested and I'm just waiting for him to get back to me atm 👍

Reply:
I think I would prefer `Io_CompanyNameModelNumber_ ` too. Makes the code for self documenting and you don't have to add sensor model to the javadoc.

Reply:
We can slowly change it for other sensors after this PR. No need to keep expanding the scope.

Reply:
ok sounds good

Reply:
I changed the file so that its in the `Io_CompanyNameModelNumber_ ` format

Reply:
On second thought I'm wondering if we should we include `BrakePressureSensor` as well as the `CompanyNameModelNumber`. Something like `Io_BrakePressureSensor_XYZ1234`

Reply:
Nah that becomes way too verbose and it breaks the existing naming scheme

Reply:
Yeah, I think that having just the model and part number is sufficient here.